bot.py

```diff
@@ -190,9 +190,20 @@ async def set_user_status():
 async def sync_whois(ctx):
     """
     Syncs the whois database with the Google sheet
+    Limited to once every 10 minutes.
     """
+    if ctx.author.guild_permissions.administrator:
```

Comment:
see https://discordpy.readthedocs.io/en/stable/interactions/api.html#discord.app_commands.checks.cooldown

Reply:
It is an app command, not a bot command. (Unless a bot can use app command APIs, then please tell me.)